docs/docs/installation_guide/installation/pre-installation/oidc-providers.md

```diff
@@ -0,0 +1,127 @@
+---
+id: oidc-providers
+title: OIDC Providers Settings
+sidebar_label: OIDC Providers Settings
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This section is for creating OIDC IdP client and creating Service Workbench root user in OIDC IdP.
+
+## Creating OIDC IdP Client
+
+OpenID Connect (OIDC) IdP is an alternative of [Cognito User Pool][cognito] for authentication of Service Workbench. Before deploying Service Workbench with OIDC IdP, you have to create your OIDC provider in external OIDC services.
+
+Below are example procedures for creating OIDC providers in Authing, Keycloak and Okta for reference. You can use other OIDC providers as well.
+
+- (Authing Option) [Authing][authing] is an example of a third-party authentication provider. 
+- (Keycloak on AWS Option) [Keycloak on AWS][keycloak-solution] is a solution maintained by AWS and can serve as an authentication identity provider.
+- (Okta Option) [Okta][okta] is one trusted platform to secure every identity, from customers to your workforce.
+
+Follow the steps below to create an OIDC client, and obtain the `client_id` and `issuer`. 
+
+### (Authing Option) Authing.cn OIDC client
+
+1. Go to the [Authing console](https://console.authing.cn/console).
+2. Create a user pool if you don't have one.
+3. Select the user pool.
+4. On the left navigation bar, select **Self-built App** under **Applications**. 
+5. Click the **Create** button.
+6. Enter the **Application Name**, and **Subdomain**.
+7. Save the `App ID` (that is, `client_id`) and `Issuer` to a text file from Endpoint Information, which will be used later.
+    [![](../../../images/OIDC/endpoint-info.png)](../../../images/OIDC/endpoint-info.png)
+
+8. Update the `Login Callback URL` and `Logout Callback URL` to your IPC recorded domain name.
```

Comment:
just put the domain name without path for callback url?

Reply:
how to configure the callback url if not using custom domain name?

Reply:
Yes, without path. I added more desciption.

Reply:
Good point, if we do not using custom domain, Service Workbench will use cloudfront domain. I will add more description for this case